src/components/user-stat-block/index.tsx

```diff
@@ -47,6 +59,10 @@ const $Content = styled.li`
     row-gap: var(--space-4);
     column-gap: 0;
   }
+  a {
+    text-decoration: none;
+    color: currentColor;
+  }
```

Comment:
For one, having a link be the exact same color as normal text makes the affordance of it almost none...

Can we keep the blue color, and maybe have the underline appear on hover?
```suggestion
  a {
    text-decoration: none;
  }
  a:hover {
    text-decoration: underline;
  }
```

Reply:
of course, I didnt know which color i should add. so i waited for the review =)